ide/src/test/java/com/devonfw/tools/ide/integrationtest/pip/PipUrlUpdaterMock.java

```diff
@@ -0,0 +1,30 @@
+package com.devonfw.tools.ide.integrationtest.python;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import com.devonfw.tools.ide.url.model.folder.UrlVersion;
+import com.devonfw.tools.ide.url.updater.pip.PipUrlUpdater;
+import com.devonfw.tools.ide.url.updater.python.PythonUrlUpdater;
+
+/**
+ * {@Link JsonUrlUpdater} test mock for Python
```

Comment:
So you also do Copy&Paste ;)
Not a JsonUrlUpdater :) and it is for Pip and not for Python ;)

Reply:
Yes, I do. Need a better workflow to not forget the javadocs when I C&P ^^.
Adjusted now.